vendor/k8s.io/kube-controller-manager/config/v1alpha1/defaults.go

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
remove this file, it should not be in `k8s.io/kube-controller-manager`, but instead in `k8s.io/kubernetes/pkg/controller/apis/config/v1alpha1`